src/v/datalake/BUILD

```diff
@@ -477,3 +477,26 @@ redpanda_cc_library(
         "@seastar",
     ],
 )
+
```

Comment:
With this PR do we have enough to remove the BAZEL_DISABLE_DATALAKE_FEATURE variable?

Reply:
I don't think so, it looks like the writer library is still switchable using an environment variable, so its still a compile dependency.